plugins/MauticFocusBundle/EventListener/ConfigSubscriber.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace MauticPlugin\MauticFocusBundle\EventListener;
+
+use Mautic\ConfigBundle\ConfigEvents;
+use Mautic\ConfigBundle\Event\ConfigBuilderEvent;
+use Mautic\ConfigBundle\Event\ConfigEvent;
+use Mautic\PageBundle\Form\Type\ConfigTrackingPageType;
+use Mautic\PageBundle\Form\Type\ConfigType;
+use MauticPlugin\MauticFocusBundle\Form\Type\FocusConfigTrackingPageType;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class ConfigSubscriber implements EventSubscriberInterface
```

Comment:
Some offensive programming won't hurt.
```suggestion
final class ConfigSubscriber implements EventSubscriberInterface
```